### ISSUE TITLE
Audio ducking is not working 

### ISSUE THREAD

**Issue by abhijitkasbe** [nvaccess/nvda#15808]
with NVDA version 23.3 version audio ducking is not working when you press NVDA+control+s you will get synthesizer dialog box pressing tab twice you will get audio ducking option if you select always duck or duck when media/speach and sound and if you press okay it will still remain on no ducking

**Comment by Adriani90** [comment]
Closing as duplicate of #15790.

**Comment by CyrilleB79** [comment]
The initial description is incomplete.

But it seems that it's rather a duplicate of #15686. To confirm this @abhijitkasbe, could you confirm that audio ducking is working with NVDA+control+D shortcut or when using the new audio settings panel (NVDA+control+U)?

If not, we may reopen this issue for more investigation.

**Comment by Adriani90** [comment]
I cannot reproduce this in NVDA last alpha. When I press ok, the setting is saved and the dialog disappears. When i open the dialog again, the setting seems activated as expected.

**Comment by seanbudd** [comment]
I'm not sure why this is re-opened.

Welcome @abhijitkasbe ,

We are unable to process this issue as it stands.

- Please fill out the issue template fully.
- Please reproduce this issue and provide a [log file](https://github.com/nvaccess/nvda/wiki/LogFilesAndCrashDumps) of the behaviour. Ensure your log level is set to debug in general preferences.

Kind Regards,
NV Access Software Developers

**Comment by Adriani90** [comment]
I pressed the wrong button I guess :)

**Comment by abhijitkasbe** [comment]
Hello!
with NVDA+control+s, if you do changes in audio ducking settings changes are not reflecting.
but as you mentioned another NVDA+control+u with that it works.
but again if you try to change from NVDA+control+s changes will remain which you have set with NVDA+control+u

**Comment by abhijitkasbe** [comment]
if you  think to re-open then let me know so , i can write in the templet.

**Comment by CyrilleB79** [comment]
@abhijitkasbe I confirm that what you describe is the same (a duplicate) of #15686.

Moreover, in last alpha (development) versions, audio ducking is gone from the synth selection dialog (NVDA+control+S) as already written in 2023.3 change log. In 2024.1, you will be forced to use the combobox in the audio panel, or its shortcut NVDA+shift+D.

Thus there is no need to reopen this issue.

**Comment by abhijitkasbe** [comment]
okay
